docs/sources/alerting/unified-alerting/opt-in.md

```diff
@@ -6,11 +6,11 @@ weight = 128
 
 # Enable Grafana 8 Alerts
 
-Setting the `ngalert` feature toggle enables the new Grafana 8 alerting system.
+Grafana 8 alerting system is enabled by default.
 
-> **Note:** We recommend that you backup Grafana's database before enabling this feature. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
+> **Note:** We recommend that you backup Grafana's database before upgrading to the next 8.2 release. If you are using PostgreSQL as the backend data source, then the minimum required version is 9.5.
 
-At startup, when [the feature toggle is enabled]({{< relref "../../administration/configuration.md">}}#feature_toggles), the legacy Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana instance.
+At startup, since [the unified alerting is enabled]({{< relref "../../administration/configuration.md">}}#unified_alerting) by default, the legacy Grafana dashboard alerting is disabled and existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana instance.
```

Comment:
```suggestion
Since [Grafana alerting is enabled]({{< relref "../../administration/configuration.md">}}#unified_alerting) by default (and legacy dashboard alerting is disabled), existing dashboard alerts are migrated into a format that is compatible with the Grafana 8 alerting system. You can view these migrated rules, alongside any new alerts you create after the migration, from the Alerting page of your Grafana instance.
```

Reply:
Move this section up so that it is the intro.